src/Tizen.NUI/src/internal/Interop/Interop.Window.cs

```diff
@@ -246,7 +246,7 @@ internal static partial class Window
             public static extern int GetNativeId(global::System.Runtime.InteropServices.HandleRef jarg1);
 
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Window_SetPositionSizeWithOrientation")]
-            public static extern void SetPositionSizeWithOrientation(global::System.Runtime.InteropServices.HandleRef jarg1, global::System.Runtime.InteropServices.HandleRef jarg2, int jarg3);
+            public static extern void ReserveSizePerOrientationForIME(global::System.Runtime.InteropServices.HandleRef jarg1, global::System.Runtime.InteropServices.HandleRef jarg2, int jarg3);
```

Comment:
could you change the names of parameters (`jarg1`, `jarg2`) into meaningful ones?

Reply:
jarg1 means window handle.
jarg2 means the reseved size per orientation.

Reply:
Should the input param name be changed  with meaningful?
I don't know this style. 
Because, the latest the other patches use as jarg1...

Reply:
done...